packer/README.md

```diff
@@ -0,0 +1,10 @@
+# Studio Selfhosted AMI instance
+
+This AMI contains K3s with preconfigured Ingress Nginx. 
+
+## Studio Installation
+To deploy Studio please:
+1. Run instance with AMI, 
+2. Open on Security Group ports: 22, 80, 443(if https is needed),
+3. Login though ssh to the instance,
+3. Proceed with the instructions of [Studio helm chart](https://github.com/iterative/helm-charts), how tu configure and install the Studio.
```

Comment:
```suggestion
## Installation
1. Create an EC2 instance with the AMI 
2. In the Security Group, allow the following ports: 22, 80, and 443 (if TLS is needed)
3. Log in to the instance with SSH
4. Proceed with the [Studio Helm chart instructions](https://github.com/iterative/helm-charts) on how to configure and install Studio.
```

Reply:
@jesper7 How about sharing the link to Notion?

Reply:
Good idea. Let's do it

Reply:
I have added a link to new docs :-) (I know they are not ready but they will be :P)